Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Shared/WindowsMixedRealityUtilities.cs

```diff
@@ -18,7 +17,7 @@
 #endif
 #endif // (UNITY_WSA && DOTNETWINRT_PRESENT) || WINDOWS_UWP
 
-namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality
```

Comment:
Adding a note here with a link to https://github.com/microsoft/MixedRealityToolkit-Unity/issues/6922 so that we capture this namespace change on this thing.

Reply:
Mostly that we should have a write up and notes about this change, because it is technically breaking (I know that other consumers that we've done work for this stuff may end up using this directly, so it's something we should call out).

@davidkline-ms  thoughts?

Reply:
Yeah, I waffled on actually changing this namespace or not. It makes it more "correct", but is a breaking change, like you pointed out. I can revert

Reply:
i have been trying, in recent PRs to document changes as they happen. even so, adding a comment to #6922 will help ensure that the change notes are as complete as possible

Reply:
Added a note to #6922